docsets/glibc/docset.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "glibc",
-    "version": "2.24",
+    "version": "2.25“,
```

Comment:
`“` isn't a regular quote symbol. It's smart quote or something like that. Notice the difference between `“` and `"`